skins/oasis/modules/OasisController.class.php

```diff
@@ -173,6 +175,9 @@ public function executeIndex($params) {
 		$this->itemType = self::getItemType();
 
 		$skin = RequestContext::getMain()->getSkin(); /* @var $skin WikiaSkin */
+		// this is bad but some extensions could have added some scripts to bottom queue
+		// todo: make it not run twice during each request
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this "TODO" comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS1135)